effekt/jvm/src/main/scala/effekt/Runner.scala

```diff
@@ -186,7 +197,7 @@ object JSNodeRunner extends Runner[String] {
 
     os match {
       case OS.POSIX =>
-        val shebang = "#!/usr/bin/env node"
+        val shebang = s"#!/usr/bin/env -S node ${flags.mkString(" ")}"
```

Comment:
Not sure how reliable `env -S` for other platforms, but I do not see another elegant way of adding the flags to the node runner. See https://unix.stackexchange.com/questions/399690/multiple-arguments-in-shebang

Reply:
I just remembered, this is very related to one of my first contributions #329, which was deemed inadequate! We should probably do something similar to #857?